### PR TITLE
feat: Add plugin to specify prettyprint class in snippets

### LINKED ISSUE #133: Code snippets don't include prettyprint class
## Expected Behavior

Snippets should have syntax highlighting.

## Actual Behavior

Syntax highlighting is not performed because the `code` HTML elements from snippets have `class="lang-cs"` or similar, but no `prettyprint` class

### PR DESCRIPTION
**Pull request by jskeet**
Fixes #133

### CODE REVIEW

**Comment by jskeet** [comment]
(Expecting the tests to fail at this point... will then add another commit with the fixed golden data.)

**Comment by jskeet** [comment]
(Not sure why the conventionalcommits check wasn't rerun though...)

**Comment by jskeet** [comment]
Urgh - using plugins then fails due to it trying to create a directory in /opt.

Will consult further in email.

File: ci/run_tests.sh [hunk 1]
```diff
@@ -21,6 +21,18 @@ export PYTHONUNBUFFERED=1
 # Add the path where pip installs commands to PATH.
 export PATH=$PATH:${HOME}/.local/bin
 
+# Copy docfx to a local directory, so that we'll be able
```
Comment (fhinkel) [inline ci/run_tests.sh:24]: This looks very hacky.

**Review by fhinkel — APPROVED** [review]

File: ci/run_tests.sh (hunk 1 above)
Comment (jskeet) [reply ci/run_tests.sh:24]: Oh it's *absolutely* hacky. I'm scoping out the boundaries of what's at least feasible :)

**Comment by jskeet** [comment]
Closed as we went with https://github.com/googleapis/doc-pipeline/pull/62

**Pull request closed without merging**